Examples/Cocoa/Sources/objc/Pin.m

```diff
@@ -307,8 +346,10 @@ - (BOOL)isEqualToPin:(Pin *)anObject
         (_counts == anObject.counts || [_counts isEqualToDictionary:anObject.counts]) &&
         (_descriptionText == anObject.descriptionText || [_descriptionText isEqualToString:anObject.descriptionText]) &&
         (_creator == anObject.creator || [_creator isEqualToDictionary:anObject.creator]) &&
+        (_tags == anObject.tags || [_tags isEqual:anObject.tags]) &&
```

Comment:
Can we use `isEqualToArray:` here?

Reply:
Good point. Not in the scope of this PR though.